### PR TITLE
Update ginkgo focus for ci-kubernetes-e2e-gce-gci-qa-serial-master

### PR DESCRIPTION
**Pull request by yujuhong**

### CODE REVIEW

**Comment by krzyzacy** [comment]
/lgtm
/shrug
good catch :-)

File: jobs/config.json [hunk 1]
```diff
@@ -2634,6 +2634,7 @@
       "--gcp-project-type=gci-qa-project",
       "--gcp-zone=us-central1-f",
       "--provider=gce",
```
Comment (wonderfly) [inline jobs/config.json]: Hmm, why not just `[Serial]` here? Unless the gce equivalent running with the same `test_args`?

Comment (krzyzacy) [reply jobs/config.json]: if you look at ci-kubernetes-e2e-gce-gci-qa-serial-m63 above, that has the right focus :-)

**Comment by wonderfly** [comment]
For changes to the `*-gci-qa-*` and `*-gci-ci-*` jobs I'd always copy @kubernetes/goog-image

**Comment by yujuhong** [comment]
> For changes to the *-gci-qa-* and *-gci-ci-* jobs I'd always copy @kubernetes/goog-image

See b/69673163

File: jobs/config.json (hunk 1 above)
Comment (yujuhong) [reply jobs/config.json]: @wonderfly you can check the config file, but it's pretty standard for `*-serial-*` suite to run these set of tests. There is no reason for this to be different.

Comment (wonderfly) [reply jobs/config.json]: Sorry I have fallen behind the config changes. Thanks for catching this. Hopefully it'll be a fix for b/69273090 as well. :)

**Pull request merged**